packages/camera/camera_android_camerax/example/android/build.gradle

```diff
@@ -1,4 +1,7 @@
 buildscript {
+    // This version should intentionally be a 1.7.* verson and lower than the 
```

Comment:
Maybe and "lower than the version of kotlin-bom..."? Some might get confused that this is a version of kotlin gradle plugin, and go look for a version of kotlin gradle plugin in `packages/camera/camera_android_camerax/android/build.gradle`, but not find one.

Reply:
Done